pkg/upstream/cluster/lb_leastconnection_test.go

```diff
@@ -53,3 +56,60 @@ func TestLACChooseHost(t *testing.T) {
 	actual = balancer.ChooseHost(nil)
 	assert.Nil(t, actual)
 }
+
+func TestLeastActiveConnectionLoadBalancer_ChooseHost(t *testing.T) {
+	verify := func(t *testing.T, info types.ClusterInfo, delta float64) {
+		bias := 1.0
+		if info != nil && info.LbConfig() != nil {
+			bias = info.LbConfig().ActiveRequestBias
+		}
+
+		hosts := createHostsetWithStats(exampleHostConfigs(), "test")
+		i := 0
+		hosts.Range(func(host types.Host) bool {
+			i++
+			h := host.(*mockHost)
+			h.w = uint32(i)
+			h.HostStats().UpstreamConnectionActive.Inc(int64(i))
+			return true
+		})
+
+		lb := newLeastActiveConnectionLoadBalancer(info, hosts)
+
+		expect := make(map[types.Host]float64)
+		actual := make(map[types.Host]float64)
+
+		hosts.Range(func(h types.Host) bool {
+			weight := h.Weight()
+			activeConnection := h.HostStats().UpstreamConnectionActive.Count()
+			expect[h] = float64(weight) / math.Pow(float64(activeConnection+1), bias)
+
+			return true
+		})
+
+		for i := 0.0; i < 100000; i++ {
+			h := lb.ChooseHost(nil)
+			actual[h]++
+		}
+
+		compareDistribution(t, expect, actual, delta)
+	}
+
+	t.Run("no bias", func(t *testing.T) {
+		verify(t, nil, 0.15)
+	})
+	t.Run("bias 0.9", func(t *testing.T) {
+		verify(t, &clusterInfo{
+			lbConfig: &v2.LbConfig{
+				ActiveRequestBias: 0.5,
```

Comment:
Why use 0.5 when the test name is "bias 0.9"?

Reply:
my mistake, renamed